### DISCUSSION TITLE
Combine all the feeds and sort by newest first in normal view?

### DISCUSSION THREAD

**Discussion by milindpatel63** [FreshRSS/FreshRSS#4143] [Q&A]
Is there a way to combine all the feeds in a single view and sort them by newest first.?

Right now, the normal view displays a specific feed sorted by newest first, then after that feed ends, the second feed begins....

**Comment by Alkarex** [comment]
Hello,
At the moment, there is no option to customise the sorting of articles. The natural order is when the articles are discovered and added to the database (just like an e-mail client) and this is by design.

When you have been using FreshRSS for a few days with a cron job regularly fetching new articles (or for WebSub-enabled feeds), you will observe that the main view is more or less what you are asking.

There is a difference between the discovery date (the one we use for sorting) and the declared publication date (the one we show on the right-hand side). The declared publication date of articles is purely informative and might be quite far in the future or in the past (typical examples include future events, academic publications) or just being missing or plain wrong, making it inappropriate for a default sorting.

See for instance this longer discussion for additional information https://github.com/FreshRSS/FreshRSS/issues/1771#issuecomment-359195655 as well as several other related issues.

We might get additional sorting options in the future, including sorting by declared publication date in addition to the discovery date, or other ways to prioritise articles in the Web UI (not in the APIs).

**Reply by milindpatel63** [reply]
Ok :(
Also, as a feature request, the card view of Fluent Reader Desktop client looks awesome. Would love a card view in the webui too.

Like This.
![image](https://user-images.githubusercontent.com/11052427/148800272-ba9e7196-7a79-4a97-ab58-86a432062d3a.png)

**Reply by Alkarex** [reply]
> Ok :(

My answer was supposed to be closer to a yes (this is what we do already) than a no ;-)

Regarding the card view, you might be able to find a desktop or mobile client you like compatible with FreshRSS: https://github.com/FreshRSS/FreshRSS/blob/edge/README.md#apis--native-apps

Otherwise the build-in thumbnails https://github.com/FreshRSS/FreshRSS/pull/3805 which might be tune by styling or extensions

**Reply by milindpatel63** [reply]
Yeah, right now Fluent Reader for Windows looks great.

**Comment by milindpatel63** [comment]
Also, Some sources don't offer full artices in their RSS feeds. Like Verge. Would love an option to set default view to webpage for these sources.

**Reply by milindpatel63** [reply]
Nevermind, someone mentioned a css trick to fetch full articles...That worked for me :)
